src/themes/xresources.rs

```diff
@@ -0,0 +1,103 @@
+use crate::errors::Error;
+use log::debug;
+use once_cell::sync::Lazy;
+use regex::Regex;
+use std::collections::HashMap;
+
+#[cfg(not(test))]
+use std::{env, path::PathBuf};
+
+#[cfg(not(test))]
+fn read_xresources() -> std::io::Result<String> {
+    use std::io::{Error, ErrorKind};
+    let home =
+        env::var("HOME").map_err(|_| Error::new(ErrorKind::Other, "HOME env var was not set"))?;
+    let xresources = PathBuf::from(home + "/.Xresources");
+    debug!(".Xresources @ {:?}", xresources);
+    std::fs::read_to_string(xresources)
+}
+
+#[cfg(test)]
+use tests::read_xresources;
+
+static COLOR_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(r"^\s*\*(?<name>[^: ]+)\s*:\s*#(?<color>[a-f0-9]{6})(?<alpha>[a-f0-9]{2})?.*$")
```

Comment:
```suggestion
    Regex::new(r"^\s*\*(?<name>[^: ]+)\s*:\s*#(?<color>[a-f0-9]{6,8}).*$")
```

You don't need to explicitly match the alpha value, if you call `hex.parse()`, the code that parses normal hex colors from normal themes will handle everything.